server/lobby.js

```diff
@@ -165,9 +161,37 @@ class Lobby {
         gameSummaries = _(gameSummaries).chain().sortBy('createdAt').sortBy('started').reverse().value();
 
         if(socket) {
-            socket.send('games', gameSummaries);
+            let filteredGames = [];
+
+            if(socket.user) {
```

Comment:
From a UX perspective, what's the case where we are sending to a specific socket but there isn't a user on that socket? In whatever that case is, they'll see no active games.